src/main/java/com/three_tech_solutions/slot_app/services/implementations/StudentServiceImpl.java

```diff
@@ -53,21 +54,22 @@ public StudentResponse createStudent(CreateStudentRequest studentDTO) {
             throw new ResponseStatusException(INTERNAL_SERVER_ERROR,"Ocurrió un error al registrar el estudiante. Intente nuevamente");
         }
         return studentMapper.toStudentResponse(student);
+
     }
 
     private void validatePlanDetail(CreateStudentRequest studentDTO) {
         //El día de pago debe ser un dato opcional (obligatorio en caso de que el tipo de pago sea dia específico) y debe ser mayor o igual a 1 y menor o igual a 31
         if (planTypeIsBeginningOfMonth(studentDTO) & studentDTO.getPaymentDay() != null) {
-            throw new ResponseStatusException(BAD_REQUEST, "No debe especificar día de pago para el plan 'Principio de mes'.");
+            throw new ResponseStatusException(HttpStatus.BAD_REQUEST, "No debe especificar día de pago para el plan 'Principio de mes'.");
         }
 
         if (planTypeIsSpecificDay(studentDTO) && paymentDayIsInvalid(studentDTO)) {
-            throw new ResponseStatusException(HttpStatus.BAD_REQUEST, "El día de pago debe ser entre 11 y 28.");
+            throw new ResponseStatusException(HttpStatus.BAD_REQUEST, "El día de pago debe ser entre 1 y 31.");
```

Comment:
Ojo, este conflicto lo resolviste mal, porque las fechas validas ahora son entre el 11 y el 28, no entre el 1 y el 31


```suggestion
            throw new ResponseStatusException(HttpStatus.BAD_REQUEST, "El día de pago debe ser entre 11 y 28.");
```